images/images.qrc

```diff
@@ -533,5 +533,6 @@
         <file>themes/qfield/nodpi/ic_chevron_up.svg</file>
         <file>themes/qfield/nodpi/ic_arrow_left_white_24dp.svg</file>
         <file>themes/qfield/nodpi/ic_opacity_black_24dp.svg</file>
+        <file>themes/qfield/nodpi/ic_common_angle_white_24dp.svg</file>
```

Comment:
Can you make that a 24 x 24 pixel SVG?

Reply:
Done. Any particular reason why we stick to 24 to 24 and not let svg library to scale as needed.

Reply:
@suricactus , for consistency of rendering:  when the svg library scales our *_24dp.svg images, we'll get the exact same scaled stroke width when our SVGs are using the same source metrics (for _24dp it's 24 x 24 pixels).